pkg/operator/controller/controller_cluster_role.go

```diff
@@ -63,12 +63,12 @@ func addDNSNameResolverPolicyRule(cr *rbacv1.ClusterRole) {
 	cr.Rules = append(cr.Rules, rbacv1.PolicyRule{
 		APIGroups: []string{"network.openshift.io"},
 		Resources: []string{"dnsnameresolvers"},
-		Verbs:     []string{"list", "watch"},
+		Verbs:     []string{"get", "list", "watch"},
 	})
 	cr.Rules = append(cr.Rules, rbacv1.PolicyRule{
 		APIGroups: []string{"network.openshift.io"},
 		Resources: []string{"dnsnameresolvers/status"},
-		Verbs:     []string{"update"},
+		Verbs:     []string{"get", "update", "patch"},
```

Comment:
So this is the update related to [the new get we do in the plugin](https://github.com/openshift/coredns-ocp-dnsnameresolver/pull/8/files#diff-0eb779b9e49d8e44b0f36923fdb8d87d5ee024f886eefc45deec4ec88380a087). However I'm wondering whether the patch verb comes from.

Reply:
It was during the implementation phase when you had [suggested](https://redhat-internal.slack.com/archives/C04VCUBSU4U/p1714060998147569?thread_ts=1713964224.175029&cid=C04VCUBSU4U) to add `get`, `update` and `patch` verbs for the subresources based on kubebuilder's suggestion. That's why I added `patch` as well in the list.